third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -1373,119 +1399,53 @@ func resourceComputeInstanceUpdate(d *schema.ResourceData, meta interface{}) err
 
 		if d.HasChange(prefix + ".subnetwork") {
 			if !d.HasChange(prefix + ".network") {
```

Comment:
nit: since there's nothing else that happens if it has a subnetwork change, I'd recommend just consolidating these two if statements with an &&

Reply:
done